lib/components/viewers/pattern-viewer.tsx

```diff
@@ -94,7 +108,7 @@ class PatternViewer extends Component {
       const operator = getRouteOperator(viewedRouteObject, transitOperators)
       const routeColor = getRouteColorBasedOnSettings(operator, route)
       const textColor = getMostReadableTextColor(routeColor, route?.textColor)
-      const fill = vehicleIconHighlight === false ? null : textColor
+      const fill = vehicleIconHighlight === false ? undefined : textColor
```

Comment:
Why change from null to undefined?

Reply:
This is to comply with the Typescript definition of the CSS `fill` attribute.